rc/qml/EventsOverview.qml

```diff
@@ -187,7 +191,14 @@ OrganicView {
     Page {
         id: sharePicker
         visible: false
-        title: i18n.tr("Share to")
+        
+        header: PageHeader{
+            
+            width: parent.width
+            height: parent.height
```

Comment:
It doesn't feel right... this way PageHeader is going to cover the content of the whole page, since it has an higher stacking value by default :)
You should remove that line, and let PageHeader use its implicitHeight instead.